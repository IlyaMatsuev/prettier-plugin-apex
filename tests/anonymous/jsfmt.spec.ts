import { fileURLToPath } from "url";

runSpec(fileURLToPath(new URL(".", import.meta.url)), ["apex-anonymous"], {
  apexExplicitAccessModifier: true,
  apexSortModifiers: true,
  astCompareDisabled: true,
});
