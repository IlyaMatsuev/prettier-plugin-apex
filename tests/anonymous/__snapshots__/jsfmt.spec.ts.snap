// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex-anonymous: AnonymousBlock.cls: AnonymousBlock.cls 1`] = `

// Leading Inline Comment in Anonymous Block

/* Leading Block Comment in Anonymous Block */

String someVariable = '';

System.debug('Hello World');
System.debug('Hi');

with sharing public class SomeClass {
    
}

// Trailing Inline Comment in Anonymous Block

/* Trailing Block Comment in Anonymous Block */
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
// Leading Inline Comment in Anonymous Block

/* Leading Block Comment in Anonymous Block */

String someVariable = '';

System.debug('Hello World');
System.debug('Hi');

public with sharing class SomeClass {
}

// Trailing Inline Comment in Anonymous Block

/* Trailing Block Comment in Anonymous Block */

`;
